lib/publish-rsr.js

```diff
@@ -0,0 +1,96 @@
+import * as Sentry from '@sentry/node'
+import timers from 'node:timers/promises'
+import pRetry from 'p-retry'
+
+const ONE_HOUR = 60 * 60 * 1000
+
+const withPgClient = fn => async ({ createPgClient, ...args }) => {
+  const pgClient = await createPgClient()
+  try {
+    return await fn({ pgClient, ...args })
+  } finally {
+    await pgClient.end()
+  }
+}
+
+export const postEvaluate = withPgClient(async ({ pgClient, round, gitCommit }) => {
+  await pgClient.query(`
+    INSERT INTO publish_rsr_rounds
```

Comment:
We usually start variable/property/table names with a noun, and function/method names with a verb.

Let's rename `publish_rsr_rounds` to follow that convention.

For example: `unpublished_rsr_rounds`, `evaluated_rsr_round`, `recent_rsr_rounds`.